### DISCUSSION TITLE
Present pop up when loading a park that using RCT1 stuff if RCT1 isn't installed.

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#17806] [Feature requests] [answered]
I think it would be a very much appreciated addition to get a pop up message when loading a park that has RCT1 stuff in it as it makes things look different. It's annoying in contests where people vote on parks but they look quite different because people don't realize RCT1 assets have been used.
It could be a simple red pop up box, or it could be a two option menu similar to save's overwrite window with "continue loading" and "cancel loading".

**Comment by duncanspumpkin** [comment]
I'm sure we already have an issue for this in the tracker but i cant seem to find it.

**Comment by Gymnasiast** [comment, marked as answer]
Implemented by https://github.com/OpenRCT2/OpenRCT2/pull/17928.
